docs/news/index.qmd

```diff
@@ -34,6 +34,7 @@ This page provides the release notes associated with each release of RStudio and
 - Improved support for development documentation when a package has been loaded via `devtools::load_all()`. (#13526)
 - Better error message when user preferences fail to save due to folder permissions. (#12974)
 - With screen reader support enabled, hitting ESC key allows Tabbing away from editor. [accessibility] (#13593)
+- Added Databricks integration to RStudio Pro Sessions which allows users to seamlessly interact with Databricks clusters and resources (rstudio-pro#4927)
```

Comment:
I still think we should move this down to the Workbench section. IIUC this section is really just for open source.

Reply:
For sure. Sorry if I didn't notice you mentioning that beforehand.